README.md

```diff
@@ -10,11 +10,13 @@
 
 This ASCOM driver provides additional functionality on top of the specified focuser device driver it connects to. At the time of this writing, the only functionality that has been implemented so far is that it averages temperature readings over 120 seconds. Indeed, the temperature reported by some focusers, like ZWO’s EAF, can be "jumpy". In order to implement temperature compensation, it is essential to have an accurate and stable temperature reading, which is what this virtual driver accomplishes. In the future, additional functionality may be implemented as needed by the author.
 
+Another example is position tolerance. Some old focusers miss one or two steps when moving. Usually, this difference is not visible in the image, but the focusing algorithm will never finish or may fail due to a timeout. In this example, we have added a focuser tolerance, and it will report the goal focus position when asked if the real driver position is within this tolerance.
```

Comment:
@rbarbera The PR looks great. However, would you mind expanding a little more on this section? I fail to understand the root of the problem.

* Which focusing algorithm are you referring to? The one implemented in NINA?
* Why would the focusing algorithm never finish?
* Which models of focuser encounter this kind of issues?

Thanks!

Reply:
Hi @jlecomte the only thing that I don't know is about other focusers with this problem. We know about the same model used by others users for example with SGP  https://forum.sequencegeneratorpro.com/t/does-sgp-talk-to-planewaves-pwi-autofocus/7027/161

I've changed all the description of the problem and how this changes fixes it. Let me know if you want more info about this change.

Regards